### DISCUSSION TITLE
Queue mod install/uninstall on server restart

### DISCUSSION THREAD

**Discussion by gnarlyvr1** [gameserverapp/Platform#571] [Suggestions]
The ability to apply mod installs/uninstalls to a server and have it take effect on the next restart would be amazing. Right now I have to coordinate with my community to shutdown and install mods. It could be a huge benefit to admins and players to install a mod, and then just announce to the community it will be installed on the next server restart.

I don't know what technical barriers there are to make that happen, but it is one feature Nitrado has over GSA currently.

Keep up the good work guys, yall are killing it!

**Comment by mvaessen** [comment]
Hi,

Thanks for your suggestion! This feature will be present in the upcoming update

**Comment by gnarlyvr1** [comment]
Awesome! Thanks!
